### PR TITLE
Fix styling bugs according to reviews

### PR DESCRIPTION
**Pull request by Sevenpros**
## Fix styling bugs 
- remove empty lines 
- fix font size and font family

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<body>
+    <header class="top-header">
+        <div class="icons-section">
+            <div class="time">
+                <img alt="time icon" src="images/time.png">
+            </div>
+            <div class="group-icons">
+                <img alt="icon of cellural network,wifi and battery" src="images/group.png">
+            </div>
+        </div>
```
Comment (muneebulrehman) [inline index.html]: This section is just a representation of how the project looks on a mobile screen, looks like you took it literally. Please remove it. 
nav-issue

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                </p>
+                <div class="social-icons">
+                    <a href="#"><img alt="github icon" src="images/git.png" class="git-icon"></a>
+                    <a href="#"><img alt="linkedin icon" src="images/Linkedin.png" class="linkedin-icon"></a>
+                    <a href="#"><img alt="social media icon" src="images/peace.png" class="social-icon"></a>
+                    <a href="#"><img alt="twitter icon" src="images/twitter.png" class="twitter-icon"></a>
+                    <a href="#"><img alt="medium icon" src="images/Vector.png" class="medium-icon"></a>
+                </div>
+            </article>
+        </section>
```
Comment (muneebulrehman) [inline index.html:42]: Your layout is still breaking on mobile screens as pointed out by the previous reviewer. Please fix it.
Here is a reference image.
issue

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi , @Sevenpros,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good linter setup and linters are passing.
- Used semantic HTML tags.

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please don't create a new PR for a re-review. This will not put me in sync with the issues raised by previous reviewers. Please keep this in mind for your future reviews. Making a new PR will result in an invalid request in the future.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sevenpros** [comment]
## Fixing the header and mobile responsiveness
- remove unnecessary icons
- fix mobile responsiveness (make the website fit mobile devices).
- 
image

**Review by OLIPLICHE — APPROVED** [review]
Hi @Sevenpros , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**